omg/core/views.py

```diff
@@ -108,8 +109,21 @@ def setup(self):
         )
 
         # Pickup button image
-        self.pickup_button_image = os.path.join(ASSET_DIR, "pickup_button", "F.png")
-        self.pickup_button_image_scale = 0.1
+        # Set grey_background
+        self.pickup_grey_background = os.path.join(
+            ASSET_DIR, "pickup_button", "grey_background.png"
+        )
+        self.pickup_grey_background_image_scale: float = 0.3
+
+        # Set button text attributes
+        pickup_key_text: str = "F"
+        pickup_key_text_font: int = 18
+        self.text_object: arcade.Text = arcade.Text(
+            pickup_key_text, 0, 0, arcade.color.BLACK, pickup_key_text_font
+        )
+        height_offset = (pickup_key_text_font // 10) * (pickup_key_text_font % 10 + 1)
+        self.text_width = self.text_object.content_width
+        self.text_height = self.text_object.content_height - height_offset
```

Comment:
should this be also wrapped in a private function (then it would be easier to just move it to a GUI class whenever we refactor the whole GUI logic out of GameView)?